src/robot/reporting/jsbuildingcontext.py

```diff
@@ -36,6 +38,8 @@ def __init__(self, log_path=None, split_log=False, prune_input=False):
         self.split_results = []
         self.min_level = 'NONE'
         self._msg_links = {}
+        expand_matcher = ExpandKeywordMatcher(expand_args)
+        self._expand_matcher = expand_matcher if expand_matcher.active() else None
```

Comment:
Do we need to create `ExpandKeywordMatcher` to check is it actually needed? Wouldn't something like this be enough:
```python
self._expand_matcher = ExpandKeywordMatcher(expand_keywords) \
        if expand_keywords else None
```
The above would remove the need to have the `active()` method. Also, `expand_args` should be renamed to something more explicit like `expand_keywords`.

Reply:
correct. improved.